torch/package/package_exporter.py

```diff
@@ -850,14 +850,6 @@ def deny(self, include: "GlobPattern", *, exclude: "GlobPattern" = ()):
 
     def _persistent_id(self, obj):
         if hasattr(obj, "__reduce_package__") and not inspect.isclass(obj):
-            if _gate_torchscript_serialization and isinstance(
```

Comment:
if we are removing this check, shouldn't we remove the `_gate_torchscript_serialization` variable as well?

Reply:
Yeah, I wanted to ask what was this for? Do we need to keep this in an internal version?